app/src/main/java/com/github/leuludyha/ibdb/di/AuthenticationModule.kt

```diff
@@ -24,10 +22,14 @@ object AuthenticationModule {
     fun provideAuthenticationContext(libraryRepository: LibraryRepository): AuthenticationContext {
         return AuthenticationContext(
             MainUser(
-                Firebase.auth.currentUser?.uid!!,
-                Firebase.auth.currentUser?.displayName ?: "username",
-                Firebase.auth.currentUser?.photoUrl.toString(),
-                Firebase.auth.currentUser?.phoneNumber,
+                //TODO: Unfortunately cannot call Firebase directly from here (would fail if there were
+                // no available network). I put mocks for now, but this should be replaced by
+                // an access to the stored values in persistent memory in my opinion once this
+                // option is available.
+                Mocks.mainUser.userId,
+                Mocks.mainUser.username,
+                Mocks.mainUser.profilePictureUrl,
+                Mocks.mainUser.phoneNumber,
```

Comment:
So using the `SharedPreferences?`

Reply:
Done in next commit. Unfortunately, super messy. I am struggling to find a clean way of implementing this. I will ask tomorrow in the stand up meeting.